scaledToFit md file and image added (#1406)
Codecademy/docs: content/swiftui/concepts/viewmodifier/terms/scaledToFit/scaledToFit.md

---
Title: '.scaledToFit()'

> Description: 'The scaledToFit() modifier can be used to scale the view to fit its parent while maintaining the original aspect ratio.'

Reviewer: ```suggestion
Description: 'Scales the view to fit its parent view while maintaining the original aspect ratio.'
```

Subjects:
  - 'Mobile Development'
  - 'Computer Science'
Tags:
  - 'SwiftUI'
  - 'SwiftUI Modifiers'
CatalogContent:
  - 'learn-swift'
  - 'paths/computer-science'
  - 'paths/build-ios-apps-with-swiftui'
---

> The **`scaledToFit()`** modifier can be used to scale the [view](https://www.codecademy.com/resources/docs/swiftui/views) to fit its parent while maintaining the original aspect ratio.

Reviewer: ```suggestion
The **`.scaledToFit()`** modifier can be used to scale the [view](https://www.codecademy.com/resources/docs/swiftui/views) to fit its parent view while maintaining the original aspect ratio.
```

## Syntax

```pseudo
struct MyStructure: View {
    var body: some View {
        Image("imagename")

>             The scaledToFit() modifier will be placed here.

Reviewer: ```suggestion
            .scaledToFit()
```

    }
}
```

## Example

In the example below, the image contains the `scaledToFit()` modifier.

```swift
import SwiftUI

struct MyStructure: View {
    var body: some View {
        Image("imagename")
            .resizable()
            .scaledToFit()
    }
}
```

> In the example above, the image that contains the `scaledToFit()` modifier has been scaled down to fit the width of the parent. The original aspect ratio of the image has been maintained.

Reviewer: ```suggestion

In the example above, the image that contains the `scaledToFit()` modifier has been scaled down to fit the width of the parent. The original aspect ratio of the image has been maintained.
```

> ![SwiftUI Modifier](https://raw.githubusercontent.com/Codecademy/docs/main/media/scaled-to-fit.jpg)

Reviewer: ```suggestion
![SwiftUI ViewModifier scaledToFit](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-viewmodifier-scaledToFit.jpg)
```

Could we crop this image to only what will be displayed on the mobile screen?
